pyvrp/tests/test_Model.py

```diff
@@ -257,3 +257,28 @@ def test_data_warns_about_scaling_issues(recwarn):
     model.add_edge(depot, client, distance=MAX_USER_VALUE + 1)
     with assert_warns(ScalingWarning):
         model.data()
+
+
+@mark.filterwarnings("ignore::pyvrp.exceptions.EmptySolutionWarning")
+def test_model_solves_instance_with_zero_or_one_clients():
```

Comment:
Can we instead check that this warning is raised for empty solutions? The filter here seems unnecessary for what is IMHO appropriate behaviour.